SKILL.tsv

```diff
@@ -4244,77 +4244,77 @@ SKILL_20150714_004243	Number of buff you can receive
+SKILL_20150714_004275	$Leather Mastery: Dexterity
+SKILL_20150714_004276	$Able to equip [Leather] armor. In addition, increases your Evasion depending on the number of equipped [Leather] armor and the attribute level. (Applies from 3 armor pieces)
+SKILL_20150714_004277	$Plate Mastery: Defense
+SKILL_20150714_004278	$Able to equip [Plate] armor. In addition, increases your maximum HP and Physical Defense depending on the number of equipped [Plate] armor and the attribute level. (Applies from 3 armor pieces)
+SKILL_20150714_004279	$Dagger Mastery
+SKILL_20150714_004280	$Deals 20 additional damage per attribute level when attacking behind the enemy with a dagger. 
+SKILL_20150714_004281	$Companion Riding
+SKILL_20150714_004282	$You can mount a Companion.
+SKILL_20150714_004283	$Increases the damage dealt on an enemy with [Bash] by 1% per attribute level.
+SKILL_20150714_004284	$Increases the damage dealt on an enemy with [Thrust] by 1% per attribute level.
+SKILL_20150714_004285	$Increases the damage dealt on an enemy with [Pommel Beat] by 1% per attribute level.
+SKILL_20150714_004286	$Increases the damage dealt on an enemy with [Double Slash] by 1% per attribute level.
+SKILL_20150714_004287	$Splash will be added onto [Bash] by 1 per attribute level.
+SKILL_20150714_004288	$Activates consecutive attacks when using [Thrust] by 1% per attribute level.
+SKILL_20150714_004289	$Double Slash: Critical Chance Increase
+SKILL_20150714_004290	$Increases the Critical Chance of [Double Slash] by 5% per attribute level.
+SKILL_20150714_004291	$Restrain: Slow
+SKILL_20150714_004292	$Due to the effects of [Restrain], stunned enemies becomes slowed after the stun is removed. Increases Slow by 3 seconds per attribute level. 
+SKILL_20150714_004293	$Increases the duration time of stun from [Restrain] by 1 second per attribute level. 
+SKILL_20150714_004294	$Increases the damage dealt on an enemy with [Rim Blow] by 1% per attribute level.
+SKILL_20150714_004295	$Increases the damage dealt on an enemy with [Umbo Blow] by 1% per attribute level.
+SKILL_20150714_004296	$Increases the damage dealt on an enemy with [Shield Lob] by 1% per attribute level.
+SKILL_20150714_004297	$Increases the damage dealt on an enemy with [Butterfly] by 1% per attribute level.
+SKILL_20150714_004298	$Increases the maximum HP of your character by 5% per attribute level while [Swash Buckling] is active. 
+SKILL_20150714_004299	$Enemies hit by [Umbo Blow] can be affected by stun with a 5% chance per attribute level. 
+SKILL_20150714_004300	$Splash will be added onto [Shield Lob] by 1 per attribute level.
+SKILL_20150714_004301	$Increases Block when equipped with a shield by 5% per attribute level.
+SKILL_20150714_004302	$After equipped with a shield, press the C button to use [Guard].
+SKILL_20150714_004303	$Enemies hit by [Umbo Blow] will fall down, increases the knockback power of by 40 per attribute level. 
```

Comment:
$Enemies hit by [Umbo Blow] will fall down, increases the knockback power *of* by 40 per attribute level.